source/lib/predicates/object.ts

```diff
@@ -5,6 +5,9 @@ import {Predicate, PredicateOptions} from './predicate';
 import hasItems from '../utils/has-items';
 import ofType from '../utils/of-type';
 import ofTypeDeep from '../utils/of-type-deep';
+import {partial, exact, Shape} from '../utils/match-shape';
+
+export {Shape};
```

Comment:
This feels a little bit weird but because it's part of the API, we need to expose this to the user as well. And because it's part of the `object` predicate, I feel like it should be exposed from within the predicate itself.

Reply:
Makes sense.